src/resources/ResourceSnapshots/ResourceSnapshots.ts

```diff
@@ -0,0 +1,15 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {ResourceSnapshotsModel} from './ResourceSnapshotsInterfaces';
+
+export default class ResourceSnapshots extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/snapshots`;
+
+    constructor(protected api: API) {
+        super(api);
+    }
```

Comment:
You can remove the constructor as you don't do anything special other than calling `super`